server/webapp/WEB-INF/rails/webpack/views/pages/users.tsx

```diff
@@ -154,6 +164,10 @@ export class UsersPage extends Page<null, State> {
     });
   }
 
+  private getMeta() {
+    return document.body.dataset.meta ? JSON.parse(document.body.dataset.meta) : {};
```

Comment:
HTMLElement.dataset. TIL.

Maybe you want to improve this: https://github.com/gocd/gocd/blob/048a28e840df8d255c1552c430312925963a581c/server/webapp/WEB-INF/rails/webpack/helpers/spa_base.tsx#L72-L84

Reply:
Done